ethers-core/src/types/chain.rs

```diff
@@ -334,6 +337,7 @@ impl Chain {
             Boba |
             ZkSync |
             ZkSyncTestnet |
+            LineaTestnet |
```

Comment:
I don't think this is right. supports eip1559:

https://explorer.goerli.linea.build/tx/0x0bfb5ade28b967cd4ab7721d624cd8c580558a5bbd2f63ceee3dd1c6312680a9